assets/sass/components/_related-topics.scss

```diff
@@ -0,0 +1,46 @@
+/* ==========================================================================
+   COMPONENTS / #RELATED TOPICS
+   ========================================================================== */
+
+.govuk-hub-related-topics {
+  background-color: govuk-colour('light-grey');
+
+  a {
+    color: govuk-colour('blue');
+
+    &:hover {
+      color: govuk-colour('dark-blue');
+    }
+
+    &:visited {
+      color: govuk-colour('purple');
+    }
+
+    &:active {
+      color: govuk-colour('black');
+    }
+  }
```

Comment:
Is this different to '.govuk-link'? If possible I'd probably push for it being the same as the standard link and use that class?

Reply:
That's correct. Using the inline .govuk-link was approach I started with. However, as the component requires a toggle between light and dark designs I don't believe this can be done using inline styles. Hence replicating .govuk-link in this case. Happy to talk solutions to this though.